genmod/score_variants/compound_scorer.py

```diff
@@ -16,13 +16,44 @@
 import sys
 import os
 import logging
-
+from typing import Dict, Tuple
 from multiprocessing import Process
 
 from genmod.vcf_tools import (replace_vcf_info, add_vcf_info)
 
+from genmod.score_variants.score_variant import as_normalized_max_min
+from genmod.score_variants.rank_score_variant_definitions import RANK_SCORE_TYPE_NAMES
+
 logger = logging.getLogger(__name__)
 
+
+def possibly_normalized_rank_score(rank_score_type: str,
+                                   threshold: int or float,
+                                   min_rank_score_value: float,
+                                   max_rank_score_value: float) -> int or float:
+    """
+    Wraps normalization logic for legacy rank_score and normalized rank_score.
+
+    Args:
+        rank_score_type: Type according to RANK_SCORE_TYPE_NAMES
+        threshold: A rank_score like value
+        min_rank_score_value: input to as_normalized_max_min
+        max_rank_score_value: input to as_normalized_max_min
+    Returns:
+        A rank_score like value, possibly normalized
+    """
+    if rank_score_type == 'RankScore':
+        # Don't perform normalization
+        return threshold
+    elif rank_score_type == 'RankScoreNormalized':
+        # Convert score into
+        return as_normalized_max_min(score=float(threshold),
+                                     min_score_value=min_rank_score_value,
+                                     max_score_value=max_rank_score_value)
+    else:
+        raise ValueError('Unknown RANK_SCORE_TYPE_NAMES config', rank_score_type)
```

Comment:
```suggestion
    raise ValueError('Unknown RANK_SCORE_TYPE_NAMES config', rank_score_type)
```
I do not think you need the else since you return in the ifs